### PR TITLE
modifer->modifier; resync->rsync; mmaped->mapped; unmarshaling->unmarshalling

### PR DESCRIPTION
**Pull request by lixin963**

### CODE REVIEW

File: consoles/node.html [hunk 1]
```diff
@@ -20,7 +20,7 @@ <h1>Node</h1>
 {{ range query "up{job='node'}" | sortByLabel "instance" }}
 <tr>
   <td><a href="node-overview.html?instance={{ .Labels.instance }}">{{ reReplaceAll "(.*?://)([^:/]+?)(:\\d+)?/.*" "$2" .Labels.instance }}</a></td>
-  <td{{ if eq (. | value) 1.0 }}>Yes{{ else }} class="alert-danger">No{{ end }}</td>
+  <td> {{ if eq (. | value) 1.0 }}>Yes{{ else }} class="alert-danger">No{{ end }}</td>
```
Comment (Nexucis) [inline consoles/node.html:23]: can you revert this changes. Excepting if you are able to explain what problems you are trying to solve, the code is fine like stated in #10923.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1464,7 +1464,7 @@ This is a breaking change to the Kubernetes service discovery.
 * [FEATURE] EC2 service discovery: Expose `ec2_instance_state`.
 * [ENHANCEMENT] Various performance improvements in local storage.
 * [BUGFIX] Zookeeper service discovery: Remove deleted nodes.
-* [BUGFIX] Zookeeper service discovery: Resync state after Zookeeper failure.
+* [BUGFIX] Zookeeper service discovery: Rsync state after Zookeeper failure.
```
Comment (Nexucis) [inline CHANGELOG.md:1467]: Here it's the correct word.

File: consoles/node.html (hunk 1 above)
Comment (lixin963) [reply consoles/node.html:23]: ok，thanks！

File: CHANGELOG.md [hunk 3]
```diff
@@ -335,7 +335,7 @@ This vulnerability has been reported by Aaron Devaney from MDSec.
 * [ENHANCEMENT] Templating: Enable parsing strings in `humanize` functions. #8682
 * [BUGFIX] UI: Provide errors instead of blank page on TSDB Status Page. #8654 #8659
 * [BUGFIX] TSDB: Do not panic when writing very large records to the WAL. #8790
-* [BUGFIX] TSDB: Avoid panic when mmaped memory is referenced after the file is closed. #8723
+* [BUGFIX] TSDB: Avoid panic when mapped memory is referenced after the file is closed. #8723
```
Comment (roidelapluie) [inline CHANGELOG.md:338]: This is corrct as is. MMAP is memory-map.

File: RELEASE.md [hunk 4]
```diff
@@ -86,7 +86,7 @@ Check the [dependencies GitHub label](https://github.com/prometheus/prometheus/l
 This bot currently does not manage `+incompatible` and `v0.0.0` in the version specifier for Go modules.
 
 Note that after a dependency update, you should look out for any weirdness that
-might have happened. Such weirdnesses include but are not limited to: flaky
+might have happened. Such weirdness include but are not limited to: flaky
```
Comment (roidelapluie) [inline RELEASE.md:89]: I think the original is valid as well.

File: consoles/node.html (hunk 1 above)
Comment (roidelapluie) [reply consoles/node.html:23]: The original is valid.
